tests/conftest.py

```diff
@@ -2273,7 +2273,10 @@ def recover_acl_rule(duthosts, enum_rand_one_per_hwsku_hostname):
     dut_tmp_dir = "/tmp"
     dut_conf_file_path = os.path.join(dut_tmp_dir, acl_rules_template)
 
-    pre_acl_rules = duthost.acl_facts()["ansible_facts"]["ansible_acl_facts"]["DATAACL"]["rules"]
+    pre_acl_rules = None
+    acl_facts = duthost.acl_facts()["ansible_facts"]["ansible_acl_facts"]
+    if 'DATAACL' in list(acl_facts.keys()):
```

Comment:
It would be more pythonic to use code like this:
```python
    pre_acl_rules = acl_facts.get("DATAACL", {}).get("rules", None)
```

Reply:
Thanks, so pythonic